components/dropdown/user/src/index.js

```diff
@@ -48,9 +49,9 @@ class DropdownUser extends Component {
     const { user, menu, expandOnMouseOver } = this.props
     const { name, avatar } = user
     const wrapperClassName = cx('sui-DropdownUser', {
-      'is-expanded': expanded
+      'is-expanded': expanded,
+      'has-notifications': this.props.menu.some(({ notifications }) => Boolean(notifications))
```

Comment:
You can use directly const `menu` because it's already deconstructed.

Reply:
ok, thanks.